frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx

```diff
@@ -453,6 +453,10 @@ export default class NativeQueryEditor extends Component {
         .setDatabaseId(databaseId)
         .setDefaultCollection()
         .update(this.props.setDatasetQuery);
+      if (this._editor && !this.props.readOnly) {
+        // HACK: the cursor doesn't blink without this intended small delay
+        setTimeout(() => this._editor.focus(), 50);
```

Comment:
This is rather terrible, so if anyone knowledge about ACE can propose a better alternative, I'm all ears!

Reply:
I'm not aware of a better fix. Unfortunately, we have another hack or two like this already in there.